Svc/FileManager/FileManager.cpp

```diff
@@ -180,7 +180,7 @@ namespace Svc {
       );
     } else {
       this->log_WARNING_HI_ShellCommandFailed(
-          logStringCommand, status
+          logStringCommand, static_cast<U32>(status)
```

Comment:
**For future review:** the use of `U32` here is probably incorrect.  Shell commands can return negative status.